### ISSUE TITLE
implement loadingDelay

### ISSUE THREAD

**Issue by hosseinmd** [hosseinmd/react-concurrent#5]
For better UX that is better sometimes don't show loading when request quickly loaded.

**Comment by majidbigdeli** [comment]
I think this is a great feature.
If the delay rate is taken as a parameter.

**Comment by hosseinmd** [comment]
Done, available in v 2.1.0.

## loadingStartdelay
## Use
```js
const app = () => {
  const [query, setQuery] = useState("a query");

  const { data, isLoading, error } = useFetching(//or useFetchingCallback
    () => fetch("http://example.com/search?query=" + query),
    [query],
    {
      // Adds a delay to loading so that loading is not shown if the api is fetched sooner
      // if you set it 1000 ms, loading not showing until 1000 ms
      loadingStartDelay: 1000,
    },
  );
};
``
